issuetracker/github/github_test.go

```diff
@@ -0,0 +1,31 @@
+package github
+
+import (
+	"fmt"
+	"testing"
+)
+
+func Test_IssueTracker_issueAPIOrigin(t *testing.T) {
+	var tests = []struct {
+		input string
+		want  string
+	}{
+		{"https://github.com/preslavmihaylov/todocheck", "https://api.github.com/repos/preslavmihaylov/todocheck/issues/"},
+		{"github.com/preslavmihaylov/todocheck/", "https://api.github.com/repos/preslavmihaylov/todocheck/issues/"},
```

Comment:
Could you also add testcases for users with numbers e.g. `user123`, users with hyphen (`user-123`), also repos with numbers & hyphens & underscores